templates/contest/moss.html

```diff
@@ -0,0 +1,87 @@
+{% extends "common-content.html" %}
+
+{% block title_ruler %}{% endblock %}
+
+{% block title_row %}
+    {% set tab = 'moss' %}
+    {% set title = contest.name %}
+    {% include "contest/contest-tabs.html" %}
+{% endblock %}
+
+{% block content_media %}
+    <style>
+        .panes {
+            display: flex;
+            flex-wrap: wrap;
+            justify-content: center;
+        }
+        .pane {
+            padding: 20px;
+        }
+    </style>
+{% endblock %}
+
+{% block content_js_media %}
+    <script type="text/javascript">
+        $(function () {
+            $('.contest-moss').click(function () {
+                return confirm('{{ _('Are you sure you want Moss the contest?') }}');
```

Comment:
"Moss" should probably be written as "MOSS" everywhere.

Reply:
Fixed in latest commit.